src/Management/src/EndpointBase/CloudFoundry/CloudFoundryEndpoint.cs

```diff
@@ -66,44 +53,41 @@ public override Links Invoke(string baseUrl)
                 return hypermediaService.Invoke(baseUrl);
             }
 
-            // TODO: The below code will be removed in 3.0
-            else
-            {
-                var endpointOptions = Options.Global.EndpointOptions;
-                var links = new Links();
-
-                if (!Options.Enabled.Value)
-                {
-                    return links;
-                }
+            throw new NotImplementedException();
```

Comment:
@hananiel what should this do now?

Reply:
@tim this should be removed as you did.

Reply:
Hold on - this now throws a `NotImplementedException`, that doesn't sound ideal